ros/src/simulation/gazebo_simulator/vehicle/vehicle_gazebo_simulation_interface/src/vehicle_gazebo_info_publisher.cpp

```diff
@@ -0,0 +1,102 @@
+#include <ros/ros.h>
+#include <std_msgs/Float64.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <gazebo_msgs/GetLinkState.h>
+#include <sensor_msgs/JointState.h>
+#include <autoware_msgs/VehicleStatus.h>
+#include <string>
+#include <cmath>
+
+class VehicleGazeboInfoPublisher
+{
+  private:
+    ros::NodeHandle nh_;
+    ros::NodeHandle pnh_;
+    ros::ServiceClient client_;
+    ros::Publisher vehicle_pose_pub_;
+    ros::Publisher vehicle_vel_pub_;
+    ros::Publisher steering_angle_pub_;
+    ros::Publisher vehicle_status_pub_;
+    ros::Timer publish_timer_; // publish timer
+    ros::Subscriber odom_sub_;
+    double wheel_radius_;
+    std::string ns_;
+    void publishTimerCallback(const ros::TimerEvent &e);
+    void jointStateCallback(const sensor_msgs::JointState::ConstPtr &input_msg);
+
+  public:
+    VehicleGazeboInfoPublisher();
+    ~VehicleGazeboInfoPublisher(){};
+};
+
+VehicleGazeboInfoPublisher::VehicleGazeboInfoPublisher() : nh_(""), pnh_("~")
+{
+    client_ = nh_.serviceClient<gazebo_msgs::GetLinkState>("/gazebo/get_link_state");
+    vehicle_pose_pub_ = nh_.advertise<geometry_msgs::PoseStamped>("/vehicle_info/pose", 1, true);
+    vehicle_vel_pub_ = nh_.advertise<std_msgs::Float64>("/vehicle_info/velocity", 1, true);
```

Comment:
velocity and steering_angle can be merged to Twist topic. In `vel_pose_connetct`, `geometry_msgs/TwistStamped` message is needed.

Reply:
Steering angle can not be merged because it is not angular velocity

Reply:
Done
ce223de115d0e234645ff8d596629a25a73f4a4b